GAE/src/com/gallatinsystems/surveyal/domain/SurveyedLocale.java

```diff
@@ -350,5 +350,14 @@ public static String generateBase32Uuid(String oldStyleIdentifier) {
                 oldStyleIdentifier.substring(4, 8),
                 base32Uuid.substring(10));
     }
-    
+
+    public void setGeoLocation(String geoLocationString) {
```

Comment:
Just in case: is it possible that geoLocationString is null?

Reply:
Ahah! good catch!